The C++ API documentation available on website does not contain information about operators in op.h

Note: Providing complete information in the most concise form is the best way to get help. This issue template serves as the checklist for essential information to most of the technical issues and bug reports. For non-technical issues and feature requests, feel free to present the information in what you believe is the best form.

For Q & A and discussion, please start a discussion thread at https://discuss.mxnet.io 

## Description
The https://mxnet.incubator.apache.org/versions/master/doxygen/files.html does not contain "op.h" that contains documentation for operators.

## Environment info (Required)


Package used (Python/R/Scala/Julia):
Referred to the C++ documentation on mxnet.io website.

The op.h is generated when MXNet is built with USE_CPP_PACKAGE=1. We might not be setting this flag when generating the doxygen files for the website. @aaronmarkham

I'm seeing a lot of errors in v1.3.x.

```
Generating XML output for namespace mxnet::cpp
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:931: warning: Found unknown command `\breif'
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:1021: warning: Found unknown command `\breif'
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:1245: warning: Found unknown command `\breif'
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:1598: warning: Found unknown command `\breif'
... lots more... plus these...
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:841: warning: Found unknown command `\prime'
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:842: warning: Found unknown command `\end'
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:880: warning: Found unknown command `\breif'
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:889: warning: Found unknown command `\begin'
...
/home/ubuntu/incubator-mxnet/docs/build_version_doc/apache-mxnet/v1.3.x/cpp-package/include/mxnet-cpp/op.h:6962: warning: Found unknown command `\mathrm'
```
Were there functional changes to op.h, or would it be possible to push the recent docs fixes back to the older branches?

--- a/docs/mxdoc.py
+++ b/docs/mxdoc.py
@@ -87,10 +87,10 @@ def generate_doxygen(app):
 def build_mxnet(app):
     """Build mxnet .so lib"""
     if not os.path.exists(os.path.join(app.builder.srcdir, '..', 'config.mk')):
-        _run_cmd("cd %s/.. && cp make/config.mk config.mk && make -j$(nproc) DEBUG=1 USE_MKLDNN=0" %
+        _run_cmd("cd %s/.. && cp make/config.mk config.mk && make -j$(nproc) DEBUG=1 USE_MKLDNN=0 USE_CPP_PACKAGE=1" %
                 app.builder.srcdir)
     else:
-        _run_cmd("cd %s/.. && make -j$(nproc) DEBUG=1 USE_MKLDNN=0" %
+        _run_cmd("cd %s/.. && make -j$(nproc) DEBUG=1 USE_MKLDNN=0 USE_CPP_PACKAGE=1" %
                 app.builder.srcdir)
 
 def build_r_docs(app):

